### DISCUSSION TITLE
How to know the total expense for x?

### DISCUSSION THREAD

**Discussion by azzamsa** [ananthakumaran/paisa#262] [Q&A]
Hi, 👋

How to calculate the total expense for x account.

Let's say I buy milk every month. Now, I wanted to know how much I spent ffor milk from 01 January 2023 to 01 January 2024.

How to do this?
The `transaction` page only show the transaction list, without the total amount.

Thank you for Paisa 🥞

**Comment by montogeek** [comment]
I think you can do this with a Sheet:
```
milkCost = cost({account=Expenses:Groceries:Milk})
```

**Reply by azzamsa** [reply]
Currently I am using this command `hledger register Expenses:Gift:Alice --begin 2023-01-01 --end 2024-01-01` which gives me better visual of the history.
